docs/apis/plugintypes/mod_data/fields.md

```diff
@@ -1,81 +1,113 @@
 ---
-title: Database fields
-tags:
-  - mod_data
-  - datafield
-  - plugintype
-  - subplugin
-documentationDraft: true
----
 
-The [Database activity](https://docs.moodle.org/en/Database_module) included with Moodle includes support for several predefined [field types](./fields.md), including text, date, and URL. It is also possible to create new field types. For example, you might like to create:
+# Database Fields for Moodle 4.4.3
+
+*This documentation is a work-in-progress. Feel free to contribute.*
```

Comment:
Please remove.

This is already a draft document, though you've removed that in your efforts.